crates/task-impls/src/events.rs

```diff
@@ -191,6 +191,13 @@ pub enum HotShotEvent<TYPES: NodeType> {
 
     /// A new high_qc has been updated in `Consensus`.
     HighQcUpdated(QuorumCertificate<TYPES>),
+
+    /// A quorum proposal has been preliminarily validated.
+    /// The preliminary checks include:
+    /// 1. The proposal is not for an old view
+    /// 2. The proposal has been correctly signed by the leader of the current view
+    /// 3. The justify QC is valid
+    QuorumProposalPreliminarilyValidated(Proposal<TYPES, QuorumProposal<TYPES>>),
```

Comment:
Do you think it'd be helpful to elucidate how this event is different from the `QuorumProposalValidated` event? Unfortunately, it looks like the `QuorumProposalValidated` event's doc comment doesn't give much information right now, so an unfamiliar reader might not be able to trivially differentiate the two events without tracing the validation paths.